### DISCUSSION TITLE
Insertion of title heading/PDF metadata title from Pandoc metadata

### DISCUSSION THREAD

**Discussion by bpj** [jgm/pandoc#10368] [Q&A]
Are the insertion of a level 1 heading containing the value of the `title` metadata field, and the insertion of the same value into PDF metadata dependent only on the used template(s)?

My use case is that I have several files which I convert to PDF both singly and concatenated into one big document. I want to have both a heading for each file which shows up in the output when concatenating and also set a PDF metadata title for the concatenated document. This can of course be achieved by having a literal literal level 1 heading in each part and shifting the heading level down when generating the concatenated document but I then lose the benefits of using `\maketitle`, the `$abstract$` variable etc. in the LaTeX template. Currently I insert the author/date/version info below the title in the body with a filter when generating the standalone parts, but is there a better way?

**Comment by jgm** [comment]
title in PDF metadata is set from the variable `title-meta`.
